ui/cockpit/src/modules/components/CancelProcessInstance/CancelProcessInstance.js

```diff
@@ -0,0 +1,324 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect, useState } from "react";
+import { Modal, Button, Form, FormControl, Table } from "react-bootstrap";
+import translate from "utils/translation";
+import { getConfig } from "utils/config";
+import { addMessage, addError, clearAll } from "utils/notifications";
+import { Notifications, ActionButton, ModalFormGroup } from "components";
+import { del, get, post } from "utils/request";
+import { useHistory } from "react-router-dom";
```

Comment:
We don't have any convention on Import order, but similar imports (e.g. utility functions) should be in one block. 
What I try to do is this order:
- React
- HOCs
- utility functions
- Components
- SCSS

but you can have your own preferences.